<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlantCard</class>
 <widget class="QWidget" name="PlantCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>45</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelPlantCost">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>47</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlantAvatar">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>11</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlantBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>45</width>
     <height>63</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* { background-image: url(:/interface/images/interface/Plant_Card.png); }</string>
   </property>
  </widget>
  <widget class="QWidget" name="labelPlantMask" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>45</width>
     <height>63</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 127), stop:1 rgba(255, 255, 255, 127));</string>
   </property>
  </widget>
  <zorder>labelPlantBack</zorder>
  <zorder>labelPlantCost</zorder>
  <zorder>labelPlantAvatar</zorder>
  <zorder>labelPlantMask</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
